src/ansys/mapdl/core/_commands/database/components.py

```diff
@@ -1,10 +1,11 @@
 """These DATABASE commands allow selected subsets of entities to be
 named as components for easy selection later on.
 """
+from typing import Any, Dict
 
 
 class Components:
-    def cm(self, cname="", entity="", **kwargs):
+    def cm(self, cname: str = "", entity: str = "", **kwargs: Dict[Any, Any]) -> None:
```

Comment:
that is a good question. I guess because the creation of these functions was an automatised process, and there are some MAPDL functions which have all their arguments optional, the author Alex K. probably decided to go for having a default empty string as default argument.

Later we haven't really performed any check on these functions again. Or at least, not systematically.

Reply:
Regarding the conflict with the default value, I would then use something like:
```
def cm(self, cname: str = "", entity: Union[Literal["VOLU", "AREA", "LINE", "KP", "ELEM", "NODE"]] = "", **kwargs: Dict[Any, Any]) -> None:
```
So we account for the default case (no value) which I believe for this command specifically it is ``NODE`` but it is not specified in the docs:
<img width="506" alt="image" src="https://github.com/ansys/pymapdl/assets/28149841/508e6149-e796-494b-9691-c335f2e12113">
<img width="480" alt="image" src="https://github.com/ansys/pymapdl/assets/28149841/65f53cf8-5e47-46ab-bc5b-232fd4c4b1a3">


If we knew for sure we could:
```
def cm(self, cname: str = "", entity:Literal["VOLU", "AREA", "LINE", "KP", "ELEM", "NODE"] = "NODE", **kwargs: Dict[Any, Any]) -> None:
```